.changeset/gentle-carrots-kneel.md

```diff
@@ -0,0 +1,7 @@
+---
+"@latticexyz/cli": patch
+"@latticexyz/world-module-erc20": patch
+"@latticexyz/world-modules": patch
+---
+
+World Modules (and contracts imported by them) can now use public library methods.
```

Comment:
I would probably separate these two changesets, with the CLI being something like 

> Added support for deploying public libraries used within modules.

Reply:
done!